pages/_app/index.jsx

```diff
@@ -68,7 +68,7 @@ class App extends NextApp {
   }
 
   handlePromiseRejection = (event) => {
-    if (event.reason instanceof UnauthorizedError) {
+    if (event.reason instanceof AuthorizationError) {
```

Comment:
there should be an or clause. If any request fails user should be immediately logged out. (The same applies for componentDidCatch method)


```
if (event.reason instanceof AuthorizationError || event.reason instanceof UnauthorizedError) {
}
```

Reply:
Why do you think so?

Reply:
Because it's data protection also the dashboard may not work properly. For example, change password won't work and so on. Basically all protected endpoints will return error and user won't see any data. There is no notification that they have been logged out.

Reply:
Thank you. I really appreciate you feedback. I will add changes appropriately.